kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/OffsetCommitType.java

```diff
@@ -0,0 +1,10 @@
+package org.axonframework.extensions.kafka.eventhandling.consumer;
+
+/**
+ * Enum to define how the consumer will handle committing offsets.
```

Comment:
Feel free to add an `@author` tag with your name, and an `@since` tag referring to 4.11.1 to the class-level JavaDoc! :-)